src/tox_travis/after.py

```diff
@@ -46,6 +46,12 @@ def travis_after():
     if not after_config_matches():
         return  # This is not the one that needs to wait
 
+    # secure environment variables are not accessible for external pull requests
+    if os.environ.get('TRAVIS_SECURE_ENV_VARS') == 'false' and \
+       os.environ.get('TRAVIS_PULL_REQUEST') != 'false':
+        print('after-all disabled for external pull requests.')
+        sys.exit(0)
+
```

Comment:
Three things:

1. I'm inclined to drop the `TRAVIS_SECURE_ENV_VARS` check. It's very cool that you found a way to do that, I'm just not convinced it's a feature worth having to anyone except us.
2. There shouldn't be any message printed. We'll just add a note to the documentation that it doesn't run on pull requests.
3. We can't `sys.exit(0)` here, or it'll override the exit status of tox, which isn't what we want unless we're reporting a new error. Just `return` to let tox do its thing.